jspsych-uil-utils.js

```diff
@@ -51,7 +51,11 @@ var uil = {};
     const CONTENT_TYPE = 'Content-Type';
     const CONTENT_TYPE_TEXT_PLAIN = 'text/plain';
 
-    const LIBRARIES = ['jspsych-uil-randomization.js']
+    const LIBRARIES = [
+        'jspsych-uil-randomization.js',
+        'libs/ua-parser.min.js', // Dependency of uil-browser
```

Comment:
If possible use 'jspsych-uil-browser' to load this library onto the uil context. I would like to keep a list of public libraries here. Alternatively, create a
```JavaScript
const PRIVATE_LIBS = ['libs/ua-parser.min.js'];
```
And load the private libraries/dependencies separately.

Reply:
Fixed in 8c6411edb6e59a20997a75f313d9dabbf3fa29fe

I've decided to use your second suggestion, as this keeps loading dependencies somewhat centralised. I think this is preferred, to keep better track of what is loaded when. (Loading orders are annoying enough to debug).
In addition, placing it in a submodule would either require code duplication or exposing private methods of ``uil``, which is less preferable in my opinion